WebRoot/js/ajax/dwt/widgets/DwtTreeItem.js

```diff
@@ -291,7 +291,15 @@ function(item) {
  */
 DwtTreeItem.prototype.getNestingLevel =
 function() {
-	return this.parent.getNestingLevel() + 1;
+	var nestingLevelCheck;
+	//increment by 2 as we can have only one aria-level(1)
```

Comment:
didn't get the comment, can we elaborate?

Reply:
update dthe comment as follows: 'the toplevel tree is zero, we can have only one level(1) heading in the page which is already defined in the banner,so initializing the toplevel with level2 for all nested folders.'